tensorzero-core/src/inference/types/mod.rs

```diff
@@ -560,9 +561,13 @@ pub struct Thought {
 
 impl RateLimitedInputContent for Thought {
     fn estimated_input_token_usage(&self) -> u64 {
-        self.text
-            .as_ref()
-            .map_or(0, |text| get_estimated_tokens(text))
+        let Thought {
+            text,
+            // We explicitly ignore these fields but list them so that this implementation is very explicit
+            signature: _,
```

Comment:
We should count tokens for the signature as well, since redacted thought blocks will only have the `signature` field set